solutions/swb-reference/integration-tests/support/resources/environments/environment.ts

```diff
@@ -5,8 +5,11 @@
 import { EnvironmentStatus } from '@aws/workbench-core-environments';
 import { AxiosResponse } from 'axios';
 import ClientSession from '../../clientSession';
-import { ENVIRONMENT_START_MAX_WAITING_SECONDS } from '../../utils/constants';
-import { poll } from '../../utils/utilities';
+import {
+  ENVIRONMENT_START_MAX_WAITING_SECONDS,
+  AVOID_THROTTLE_WAITING_MILISECONDS
```

Comment:
Thoughts on calling this `DEFLAKE_DELAY_IN_MILLISECONDS`? Looks like we had a similar concept in V1 shown [here](https://github.com/awslabs/service-workbench-on-aws/blob/9e224ff1aee34ccbdea14f330c7c840a9a39b125/main/integration-tests/support/resources/base/resource.js#L129). It's interesting in both cases we arrived at 2 seconds for the delay.